### PR TITLE
Who Disclaimer

### PR DESCRIPTION
**Pull request by brett-stephen**
## Briefly describe the feature or bug that this PR addresses.
Adds WHO logo and disclaimer to explore tab on desktop and mobile

## Please link the Airtable ticket associated with this PR.
- No ticket ATM

Mobile View:
image

Desktop View:
image

### CODE REVIEW

**Comment by rahularoradfs** [comment]
Thanks, Brett! A couple of quick questions here:
- Have we added the WHO Logo to our about page at all? Would be great to see what that looks like as well.
- I noticed that the sidebar text on mobile and desktop doesn't appear to be consistent, with the desktop being the most recent. Is three any way to ensure that these are consistent?

Of course, we should hold of on merging here until we have final approval from WHO. I've just emailed them to ask they re-review.

File: src/utils/translate/en.json
Comment (atatmaja) [inline src/utils/translate/en.json:448]: If we got these from the WHO can we ask them for a french version?

**Comment by atatmaja** [comment]
> Thanks, Brett! A couple of quick questions here:
> 
> * Have we added the WHO Logo to our about page at all? Would be great to see what that looks like as well.
> * I noticed that the sidebar text on mobile and desktop doesn't appear to be consistent, with the desktop being the most recent. Is three any way to ensure that these are consistent?
> 
> Of course, we should hold of on merging here until we have final approval from WHO. I've just emailed them to ask they re-review.

Yeah re the text consistency it's not just the font but also the mobile version of the WHO disclaimer seems to be centered while the desktop version seems to be left aligned.

**Review by atatmaja — COMMENTED** [review]
Code itself LGTM but will hold on approving until we get approval from WHO

**Review by atatmaja — APPROVED** [review]
LGTM

**Pull request merged**
